### ISSUE TITLE
how to train Stable Diffusion Reference Only Automatic Coloring 0.1.2 ?

### ISSUE THREAD

**Issue by D-Mad** [aihao2000/stable-diffusion-reference-only#1]
Did you train on lora or diffuser?

**Comment by aihao2000** [comment]
@D-Mad I did not use lora training, but used the huggingface diffusers library, and the general diffusion loss function.

**Comment by aihao2000** [comment]
@D-Mad Training can use the script file under datasets to build the dataset. It requires a ```meta_data.json``` in the root of the data folder, tagging similar images. Then use the accelerate library and train/train_sd_reference_only_automatic_coloring.py to train. For parameters, please refer to train/tran_automatic_coloring_0.1.x.sh. The current training script will use the adaptive threshold of cv2 to obtain line art of similar pictures for training. I'm currently working on my paper, and I'll write a complete training tutorial in a few days.

**Comment by aihao2000** [comment]
@D-Mad I've added a training tutorial, hope it's clear enough. If you have any questions, feel free to contact me.

**Comment by D-Mad** [comment]
thank you very much ,If I train on a real people data set, will it work?

**Comment by aihao2000** [comment]
@D-Mad In theory it is possible, as long as you have a training dataset with the same accuracy.
